docker/Dockerfile.new

```diff
@@ -1,5 +1,6 @@
 ARG SGX_MODE=HW
```

Comment:
should we also organize the name of `Dockerfile`s in `docker` folder? It is growing a bit messy now.